euphorichubbbb.md

```diff
@@ -0,0 +1,43 @@
+
+The projects I build and the art I make are all explorations of ideas that interest me, and ways to learn new skills. Since much of what I do is built on the work of others
+## Budget
+
+| Product         | Supplier/Link                         | Cost   |
+| --------------- | ------------------------------------- | ------ |
+| Audio Technica ATH-50x Headphones   |https://arcus-www.amazon.in/Audio-Technica-ATH-M50x-Over-Ear-Professional-Headphones/dp/B00HVLUR86 | $151 (deliver charge included)  |
+|Arduino Micro | https://robu.in/product/original-arduino-micro-no-headers/?gclid=CjwKCAiAqaWdBhAvEiwAGAQltjwJuJy3di_P510ILxbI7BXjyh3kCLHJ0avgW-L0lOxNmu-EgVFexRoCS9gQAvD_BwE  | $19.95 |
+|        Protoboard  | https://robu.in/product/zy-208-3220pts-breadboard-4pcs-830pts-mb-102/?gclid=CjwKCAiAqaWdBhAvEiwAGAQltkMGepM2BIH85uo43mMfxx0ffWr6soxKKg-Qrf6K7fvAtGnr4wy6BRoCNyIQAvD_BwE  | $13.66 |
+|9v Battery with holder and switch| https://www.amazon.in/Electronicspices-Battery-Holder-Switch-holder/dp/B08L57WTNN| $1.08
+|Female Headers|https://www.amazon.in/Robo-India-Header-Relimate-Connector/dp/B00RUNYE66|$2.72
+|Lm384 and IC Socket| https://www.amazon.in/Audio-Amplifier-Romtek-Matching-sockets/dp/B07KYDNQ3J| $5.40
+|Electrolic and Ceramic Capacitors|https://robu.in/product/10-uf-25v-through-hole-electrolytic-capacitor-pack-of-40/ and https://robu.in/product/100nf-0603-surface-mount-multilayer-ceramic-capacitor-pack-of-40/|$1.82
+|Resistors|https://www.amazon.in/Resistors-Assortment-Tolerance-Excellently-Breadboards/dp/B07D2Z45CG?th=1| $29
+|Jumper Wires|https://robu.in/product/20cm-dupont-wire-color-jumper-cable-2-54mm-1p-1p-female-female-40pcs/?gclid=CjwKCAiAqaWdBhAvEiwAGAQltkDX_Bzh786v2ejrurNHUXmR1gjxpdL3Tiz1SD5i78Rpq36leV-8kxoCtZQQAvD_BwE| 0.60$
```

Comment:
@euphorichubbbb do you need male jumper wires instead? also, since you have a breadboard i just realised you might not need to solder it together since your breadboard just uses pins? or do you plan on soldering it later on? if so, do you need wires and a soldering-friendly protoboard?

Reply:
I have both.

Reply:
> 

great!